### PR TITLE
Release 1.29.0

### PR DESCRIPTION
**Pull request by khorne3**
## Release checklist

- [x] All feature documentation reviewed and merged
- [x] Changelog completed
- [x] Docs owner review
- [x] Release manager review
- [x] Create branch for prior versions and update `manifest.json`

### CODE REVIEW

File: guides/admin/wp-cli.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
```
Comment (khorne3) [inline guides/admin/wp-cli.md:1]: @code-asher can you please take a look at this doc?

File: guides/admin/wp-cli.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+description: Learn how to provision a workspace provider using the Coder CLI.
+---
+
+1. Install and authenticate the Coder CLI.
+
+1. Run the following to provision a new workspace provider (be sure to replace
+   the placeholders as necessary):
+
+   ```console
+   coder providers create [name] --hostname=[hostname] --cluster-address=[clusterAddress]
```
Comment (code-asher) [inline guides/admin/wp-cli.md:12]: This looks like it is creating a Kubernetes provider so we need to add `kubernetes` sub-command.  Also, I think the `--hostname` flag does not exist.  The minimum required flags are `--cluster-address` and `--namespace` so we could use those.

```suggestion
   coder providers create kubernetes [name] --namespace=[namespace] --cluster-address=[clusterAddress]
```

Comment (code-asher) [reply guides/admin/wp-cli.md:12]: One thought is that  we could have both a Kubernetes section and an EC2 section.  The minimal EC2 command is:

``` 
coder providers create ec2 [name] --access-key-id=[access-key-id] --secret-access-key=[secret-access-key]
```

File: guides/admin/wp-cli.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+   ```
+
+    <!-- markdownlint-disable -->
+
+   | **Parameter**   | **Description**                                                                |
+   | --------------- | ------------------------------------------------------------------------------ |
+   | Name            | The name for the workspace provider you'd like provisioned                     |
+   | Hostname        | The hostname of your Coder deployment                                          |
```
Comment (code-asher) [inline guides/admin/wp-cli.md:20]:
```suggestion
   | Namespace       | Namespace in which to provision workspaces. |
```

File: guides/admin/wp-cli.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+   | Cluster address | The address of the Kubernetes control plane. Find using `kubectl cluster-info` |
+
+    <!-- markdownlint-restore -->
+
+   Example usage:
+
+   ```console
+   coder providers create my-provider --hostname=https://provider.example.com --cluster-address=https://255.255.255.255`
```
Comment (code-asher) [inline guides/admin/wp-cli.md:28]:
```suggestion
   coder providers create kubernetes my-provider --namespace=my-namespace --cluster-address=https://255.255.255.255`
```

File: guides/admin/wp-cli.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+   provided `helm upgrade` command; it should look something like the following,
+   but with the placeholders filled with values appropriate to your deployment:
+
+   helm upgrade coder-workspace-provider coder/workspace-provider \
+    --version=<version> \
+    --atomic \
+    --install \
+    --force \
+    --set envproxy.token=<token> --set envproxy.accessURL=<envproxyAccessURL>
+   --set ingress.host=<ingressHostName> \
```
Comment (code-asher) [inline guides/admin/wp-cli.md:44]:
```suggestion
    --set envproxy.token=<token> \
    --set envproxy.accessURL=<envproxyAccessURL> \
    --set ingress.host=<ingressHostName> \
```

File: guides/admin/wp-cli.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+    <!-- markdownlint-restore -->
+
+   Example usage:
+
+   ```console
+   coder providers create my-provider --hostname=https://provider.example.com --cluster-address=https://255.255.255.255`
+   ```
+
+1. Once you've provisioned the workspace provider,
```
Comment (code-asher) [inline guides/admin/wp-cli.md:31]: The Helm part is beyond me unfortunately so I will defer here.

**Review by presleyp — APPROVED** [review]

**Pull request merged**